content/en/service_management/workflows/track.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Track workflows
+disable_toc: false
+algolia:
+  tags: ['workflow', 'workflows', 'workflow automation']
+further_reading:
+- link: "/service_management/workflows/build"
+  tag: "Documentation"
+  text: "Build workflows"
+- link: "/account_management/audit_trail/"
+  tag: "Documentation"
+  text: "Datadog Audit Trail"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Workflow Automation is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+This page explains how to track different kinds of workflow activity and workflow costs.
+
+## Out-of-the-box dashboard
+
+The Workflows Overview dashboard provides a high-level overview of your Datadog workflows and executions. To find the dashboard, go to your [Dashboard list][2] and search for **Workflows Overview**.
+
+{{< img src="service_management/workflows/workflows-dashboard.png" alt="The Workflows Overview dashboard" style="width:80%;" >}}
+
+
+## View editors
```

Comment:
I think it's odd that this is scoped to only "viewing editors" when you can see a lot more info than that. I think I prefer something like "View workflow edits & runs in Audit Trail"

Reply:
Sure, that's an easy change. I was looking at it from a task perspective, but your suggestion makes sense